packages/connectivity/src/scp-cf/destination/destination-service.spec.ts

```diff
@@ -194,6 +196,23 @@ describe('destination service', () => {
     });
   });
 
+  describe('fetchCertificate', () => {
```

Comment:
The test just shows the happy path. I think we should also test the error case, i.e. error is logged but not thrown.

Reply:
tests added